cvat/apps/engine/media_extractors.py

```diff
@@ -667,7 +667,7 @@ def save_as_chunk(self, images, chunk_path):
                 output = io.BytesIO()
                 if self._dimension == DimensionType.DIM_2D:
                     pil_image = rotate_within_exif(Image.open(image))
-                    pil_image.save(output, format=pil_image.format if pil_image.format else ext or self.IMAGE_EXT, quality=100, subsampling=0)
+                    pil_image.save(output, format=pil_image.format if pil_image.format else self.IMAGE_EXT, quality=100, subsampling=0)
```

Comment:
``ext`` can not be used as Pillow format. 
This test also helped to find this issue